graphql/Dockerfile

```diff
@@ -0,0 +1,60 @@
+# Build Stage
+# =============================================================================
+FROM python:3.9-slim as builder
+
+WORKDIR /app
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+# Install python app requirements
+COPY requirements.txt .
+RUN pip wheel --no-cache-dir --wheel-dir /app/wheels -r requirements.txt
+
+
+# Final Stage
+# =============================================================================
+FROM python:3.9-slim
+
+# Set metadata information
+LABEL application.name="ORCA-GraphQL" \
+    application.description="GraphQL server for ORCA application." \
+    application.team="ORCA" \
+    application.version="v0.0.16"
```

Comment:
We could make version a variable here so we pull it from the environment so that we would just pass the version in your bash script.

for example, if the shell environment variable was VERSION,  `application.version="v0.0.16"` would become `application.version="$VERSION"`

Reply:
Tried this. It does not pull from shell variables.

Reply:
Run `export VERSION_NUMBER=0.0.20` and switched tried both `application.version="$VERSION_NUMBER"` and `application.version="$VERSION_NUMBER"`
Both equated to empty strings.

Reply:
Changed